src/main/java/racingcar/model/Cars.java

```diff
@@ -1,45 +1,50 @@
 package racingcar.model;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.ArrayList;
+import java.util.List;
 
 import racingcar.Main;
 import racingcar.util.Input;
 
-public class Cars {
-	private Map<Integer, Car> cars;
+public final class Cars {
+	private static List<Car> cars;
```

Comment:
```suggestion
	private List<Car> cars;
```
상태 값을 가지는 인스턴스 변수는 static 없이 구현

클래스 변수/인스턴스 변수
클래스 메서드/인스턴스 메서드

위 두 가지를 각각 어떤 경우에 사용하는 것이 적합한지를 학습한 후 구현할 것을 추천